### PR TITLE
Link to custom-elements-everywhere.com

### PR DESCRIPTION
**Pull request by atilacamurca**
The site <https://custom-elements-everywhere.com> runs a suite of tests against a framework to
identify interoperability issues, and highlight potential fixes already implemented in other frameworks.

This is a great way to give more confidence while choosing Vue.

### CODE REVIEW

**Comment by phanan** [comment]
I'm not sure about this TBH. Time to ping @yyx990803 I guess 😅

File: src/v2/guide/index.md [hunk 1]
```diff
@@ -393,6 +393,8 @@ You may have noticed that Vue components are very similar to **Custom Elements**
 
 2. Vue components provide important features that are not available in plain custom elements, most notably cross-component data flow, custom event communication and build tool integrations.
 
+According to <https://custom-elements-everywhere.com/#vue>, Vue has a great interoperability with **Custom Elements**.
```
Comment (yyx990803) [inline src/v2/guide/index.md]: We can provide a bit more context:

"Although Vue doesn't use custom elements internally, it has great interoperability when it comes to consuming or distributing as custom elements. [According to ...]. Vue CLI also supports building Vue components that register themselves as native custom elements."

File: src/v2/guide/index.md [hunk 2]
```diff
@@ -393,6 +393,11 @@ You may have noticed that Vue components are very similar to **Custom Elements**
 
 2. Vue components provide important features that are not available in plain custom elements, most notably cross-component data flow, custom event communication and build tool integrations.
 
+Although Vue doesn't use custom elements internally, it has great interoperability when it comes to
```
Comment (phanan) [inline src/v2/guide/index.md]: Can you merge the three lines?

File: src/v2/guide/index.md [hunk 3]
```diff
@@ -393,6 +393,11 @@ You may have noticed that Vue components are very similar to **Custom Elements**
...
+consuming or distributing as custom elements. According to <https://custom-elements-everywhere.com/#vue>,
```
Comment (phanan) [inline src/v2/guide/index.md]: In order to keep the words concise while still maintaining the same information, I'd suggest something like:

> Although Vue doesn't use custom elements internally, it has [great interoperability](https://custom-elements-everywhere.com/#vue) when it comes to consuming or distributing as custom elements. Vue CLI also supports building Vue components that register themselves as native custom elements.

**Comment by phanan** [comment]
LGTM, thanks!

**Pull request merged**